모의고사.js

```diff
@@ -0,0 +1,50 @@
+function solution(answers) {
+    const QUIZLEN = answers.length;
+    // 문제 길이
+    const firstPattern = [1, 2, 3, 4, 5];
+    const FIRST = Array(QUIZLEN)
+        .fill()
+        .map((item, index) => (index % firstPattern.length) + 1);
+    // FIRST >> 1번이 쓴 답지
+    const secondPattern = [2, 1, 2, 3, 2, 4, 2, 5];
+    const SECOND = Array(QUIZLEN)
+        .fill()
+        .map((item, index) => secondPattern[index % secondPattern.length]);
+    // 2번이 쓴 답지
+    const thirdPattern = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5];
+    const THIRD = Array(QUIZLEN)
+        .fill()
+        .map((item, index) => thirdPattern[index % thirdPattern.length]);
```

Comment:
```suggestion
    const firstPattern = [1, 2, 3, 4, 5];
    const secondPattern = [2, 1, 2, 3, 2, 4, 2, 5];
    const thirdPattern = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5];

    const FIRST = Array(QUIZLEN)
        .fill()
        .map((item, index) => (index % firstPattern.length) + 1);
    // FIRST >> 1번이 쓴 답지
    const SECOND = Array(QUIZLEN)
        .fill()
        .map((item, index) => secondPattern[index % secondPattern.length]);
    // 2번이 쓴 답지
    const THIRD = Array(QUIZLEN)
        .fill()
        .map((item, index) => thirdPattern[index % thirdPattern.length]);
```
개인적으로 이게 더 가독성이 좋아보이네요